tests.html

```diff
@@ -138,13 +139,14 @@ <h2 id="g1-title"></h2>
                         WHERE TestRuns.time > ${since.getTime() / 1000} AND TestRuns.time < ${to.getTime() / 1000}`);
                 labels.unshift(to.getDate() + "." + (to.getMonth() + 1));
                 points.unshift(failed[0].values[0]);
+                limit_points.push(1);
```

Comment:
That seems too coarse. Limit should be per measurement, i.e. queue time vs. run time vs. failure rate. Current limits [are here](https://github.com/cockpit-project/cockpit/wiki/DevelopmentPrinciples#service-level-objectives)

Reply:
This is branch for failure rate, so the limit is for this metric. I know it is a bit stupid to do it like this, but I would say good enough for now :) (I am bumping it to 10% though according to what we picked)

Reply:
Ah, ok -- the gray line in "Average time waiting in queue" is also only a few seconds (possibly 1?). I think setting that to something like 15 minutes seems reasonable.

Reply:
yeah, I think I put there 1s, which was just random number.
15 minutes seems to long to me, especially since we want `Test results are available within 60 minutes in 95% of runs`. Maybe then 5 minutes? Or I'll just ditch this metric and rather have how long it took, not how long it waited (but waited indicates length of queue and availability of CI)

Reply:
I think *recording* both the queue and test run length is important -- these are both useful SLIs for figuring out where the problems are. I'm less sure about the graph -- should it just show the queue+run sum, or two graphs with the individual metrics. Is it possible to draw more than one curve into the graph, and use different scales? (seconds for queue time, minutes for run time)?

But I suppose we also don't need to overthink this. We want to know about the outliers which spend > 5 mins in the queue or > 1 hour in the test, and show the precise log and time stamp for investigation. So maybe a table is good enough for that as well.

But all of that is fungible, let's not block the data collection and initial PR on that. I'd say, for this PR just set it to 5 mins, yes.